app/frontend/packs/other-qualifications-subject-autocomplete.js

```diff
@@ -0,0 +1,26 @@
+import { accessibleAutocompleteFromSource } from "./helpers.js";
+
+const initOtherQualificationsSubjectAutocomplete = () => {
+  try {
+    const inputIds = [
+      "candidate-interface-other-qualification-wizard-subject-field",
+      "candidate-interface-other-qualification-wizard-subject-field-error",
+    ];
+
+    inputIds.forEach(inputId => {
+      const input = document.getElementById(inputId);
+      if (!input) return;
+
+      const containerId = "subject-autocomplete-data";
+      const container = document.getElementById(containerId);
+      if (!container) return;
+
+      accessibleAutocompleteFromSource(input, container);
```

Comment:
I get a JS error here if I don't select A or AS level `SyntaxError: JSON Parse error: Unexpected identifier "undefined"`. I guess this is because the data `div` is empty in this case.

Reply:
Good spot, fixing...